jsk_fetch_robot/jsk_fetch.rosinstall

```diff
@@ -9,3 +9,7 @@
 - git:
     local-name: FA-I-sensor
     uri: https://github.com/RoboticMaterials/FA-I-sensor.git
+- tar:
```

Comment:
tar! I did not know this syntax!!!
```
git:
   uri: http://github.com/fetchrobotics-gbp/fetch_ros-release/
   verison:  debian/ros-indigo-fetch-moveit-config_0.7.13-0_trusty
   local-name: fetch_ros/fetch_moveit_config
```
and also please add comment within `.rosinstall` something like
```
When ``fetch_moveit_config version 0.7.13` is released, this PR can be purged. Please check 
http://packages.fetchrobotics.com/ros/ubuntu/pool/main/r/ros-indigo-fetch-moveit-config/
```
so that every one can understand how to decide when we can remove this

Reply:
you can make this line with
```
rosinstall_generator fetch_moveit_config --rosdistro indigo --tar
```

Thank you for your review.
I will add comment out.